### ISSUE TITLE
Creatives for Social

### ISSUE THREAD

**Issue by ghost** [RashikaBTN/Digital-Marketing-Team#99]
@JigneshSolankiBTN pls help me with the creatives for USA Beer Ratings winner - I have attached 5 bottle shots of Aktien Brewery of Kaufbeuren company. You can find their names here. https://usabeerratings.com/en/competition-global-results/2020/

@KartikDesaiBTN FYI
![ABK Hell Das Blaue](https://user-images.githubusercontent.com/26758252/102443734-72589700-404d-11eb-89cb-47be78264b89.png) - 
![ABK Pils](https://user-images.githubusercontent.com/26758252/102443735-72f12d80-404d-11eb-83d8-5ad4cc263d0c.jpg)
![ABK Rose Frisch](https://user-images.githubusercontent.com/26758252/102443736-72f12d80-404d-11eb-9741-bfb9f5031091.jpg)
![ABK Schwarz Lagerbier](https://user-images.githubusercontent.com/26758252/102443738-7389c400-404d-11eb-8866-1114ca1e6561.png)

![ABK Edel Spezialbier](https://user-images.githubusercontent.com/26758252/102443733-71276a00-404d-11eb-9d18-14d665d88bdb.png)

**Comment by ghost** [comment]
@chinkabtn  Instagram size and do i have to keep all bottles in one image ?

**Comment by ghost** [comment]
Yes. Instagram size. As they're from Germany so you can give a title like Top 5 beers from Germany To Stock, German beers brands to watch in 2021, etc.. 


Here are a few examples of how I create. 
https://www.instagram.com/p/CI2Fcqdn2Ct/ 
https://www.instagram.com/p/CIn2wIkH01X/
https://www.instagram.com/p/CInf1ApHrSp/
https://www.instagram.com/p/CIg6IgQIS4V/

**Comment by JigneshSolankiBTN** [comment]
![01](https://user-images.githubusercontent.com/47557561/102453656-a7bbaf80-4062-11eb-8cf4-f9da55f87dbb.jpg)

**Comment by JigneshSolankiBTN** [comment]
please let me know changes or suggestions :)

**Comment by ghost** [comment]
![02](https://user-images.githubusercontent.com/47557561/102453949-30d2e680-4063-11eb-9164-b87c3a71fec6.jpg)

 @chinkabtn @KartikDesaiBTN Please let me know the changes or suggestions

**Comment by ghost** [comment]
![02](https://user-images.githubusercontent.com/47557561/102454398-f6b61480-4063-11eb-8594-fc870c38ca16.jpg)

**Comment by ghost** [comment]
nice thanks 👍

**Comment by ghost** [comment]
Hi @JigneshSolankiBTN @chinkabtn 
Can you please keep the german flag below the title " Germab beers to watch out..."
Or maybe even between the title like
German Beers
"insert flag"
To Watch in 2021

Whatever works


As the german flag is right next to our name

**Comment by ghost** [comment]
Right @KartikDesaiBTN 
@JigneshSolankiBTN pls change it

**Comment by ghost** [comment]
@KartikDesaiBTN  I have another option, working on changes

**Comment by ghost** [comment]
![01](https://user-images.githubusercontent.com/47557561/102457074-2b2bcf80-4068-11eb-8cfa-ebcd40e43a4e.jpg)
@KartikDesaiBTN @chinkabtn

**Comment by ghost** [comment]
Perfect, thank you @JigneshSolankiBTN
